lib/bootloader_pvm.pm

```diff
@@ -124,13 +124,18 @@ sub enter_netboot_parameters {
     }
     my $ntlm_p = get_var('NTLM_AUTH_INSTALL') ? $ntlm_auth::ntlm_proxy : '';
     type_string_slow "linux $mntpoint/linux vga=normal $ntlm_p install=$mirror ";
-    bootmenu_default_params;
+    if (get_var('AGAMA')) {
```

Comment:
I mean that you need to merge this code with the line above, there is no need to add `install=` it is the same approach for s390x, it is one or the other.

Reply:
f6d1b7e6f